src/MangaJet/app/src/main/java/com/mangajet/mangajet/log/Logger.kt

```diff
@@ -0,0 +1,66 @@
+package com.mangajet.mangajet.log
+
+import com.mangajet.mangajet.data.StorageManager
+import java.io.File
+import java.io.IOException
+import java.util.*
+import java.util.logging.FileHandler
+import java.util.logging.Level
+import java.util.logging.Logger
+import java.util.logging.SimpleFormatter
+
+// Our logger implementation
+object Logger {
+    //Create logger
+    private var l = Logger.getLogger(javaClass.toString())
+    const val fileLogPath = "/LogFiles"
```

Comment:
Bad idea i think.
Use StorageManager API instead of doing "Files" operations yourself.
Do like Librarian with files and strings